src/main/java/uk/gov/hmcts/reform/professionalapi/controller/request/validator/impl/OrganisationByProfileIdsRequestValidatorImpl.java

```diff
@@ -0,0 +1,13 @@
+package uk.gov.hmcts.reform.professionalapi.controller.request.validator.impl;
+
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.reform.professionalapi.controller.request.InvalidRequest;
+
+@Component
+public class OrganisationByProfileIdsRequestValidatorImpl {
```

Comment:
OrganisationByProfileIdsRequestValidatorImpl -> change got OrganisationByProfileIdsRequestValidator

Impl naming convention for interface implementations

Reply:
done